test/e2e/agent_dir/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.5'
+
+services:
+  agent:
+    build:
```

Comment:
perhaps the Docker file could be removed, and use directly the image here mounting the needed files as volumes. I think it would reduce complexity and execution time

Reply:
Good catch. Changed on https://github.com/newrelic/nri-docker/pull/128/commits/d6a2322a75d686fd9dcfa71bed1c4c61fb4dc669